cli.js

```diff
@@ -67,6 +67,8 @@ lighthouse({
   } else {
     prettyPrint(results);
   }
+}).catch(function () {
+  process.exit();
```

Comment:
I think this is due to the driver attempting to interact with the page and it just waiting to timeout. (then exits).

I think we could put an initial check in the CLI to fast fail if the URL does not resolve.